Add relic XP view to overlay, refactor relic XP view

alydevs opened the pull request:
Also changes the relic XP view to the Score view from stage 9/cosmic tools stage at stage 14 to support Stellar tools capping.

This feature is a bit broken at the moment (the overlay doesn't refresh unless the plugin is unloaded/reloaded and I don't know why), I don't quite know normal practices for ImGui, please test and fix bugs before adding to release

<img width="307" height="390" alt="image" src="https://github.com/user-attachments/assets/427b7d2c-8ed4-4f20-96c2-ee0467b39974" />

<img width="311" height="243" alt="image" src="https://github.com/user-attachments/assets/a54c1ddd-1a4c-46fb-904d-74c7e2ab61b9" />

alydevs committed:
Add relic XP view to overlay, refactor relic XP view

alydevs committed:
Merge branch 'Development_Main' into feat/overlayxp

LeontopodiumNivale14 commented:
Looking good! Code wise atleast. Gonna boot it up in game rq just to make sure everything is fine and dandy.

LeontopodiumNivale14 commented:
Also... see what you mean about the gui. That's an oddity in itself on why it's not refreshing *-sighs-*

LeontopodiumNivale14 committed:
Tweaked the overlay window

Originally, it was going off of what your selected in the ice menu. Now it'll display the current job you're on.

LeontopodiumNivale14 commented:
<img width="339" height="383" alt="image" src="https://github.com/user-attachments/assets/b7e85ac9-b4cc-4b43-9b56-8800079aa95d" />

<img width="343" height="421" alt="image" src="https://github.com/user-attachments/assets/ddd6c011-e2ed-4a47-9fd4-eec35b472274" />
